File: app/src/main/java/workshop1024/com/xproject/controller/activity/AboutActivity.java
package workshop1024.com.xproject.controller.activity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import workshop1024.com.xproject.R;

public class AboutActivity extends AppCompatActivity {
    private Toolbar mToolbar;

    public static void startActivity(Context context) {
        Intent intent = new Intent(context, AboutActivity.class);
        context.startActivity(intent);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        //TODO 标题的代码是否可以抽象？？
        mToolbar = findViewById(R.id.about_toolbar_navigator);
        setSupportActionBar(mToolbar);

        ActionBar actionBar = getSupportActionBar();
        actionBar.setDisplayHomeAsUpEnabled(true);

        setTitle("About Paperboy");
    }
}

File: main/src/main/java/workshop1024/com/xproject/main/about/AboutActivity.kt
package workshop1024.com.xproject.main.about

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import workshop1024.com.xproject.main.R
import workshop1024.com.xproject.main.databinding.AboutActivityBinding

class AboutActivity : AppCompatActivity() {

    companion object {
        fun startActivity(context: Context) {
            val intent = Intent(context, AboutActivity::class.java)
            context.startActivity(intent)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val aboutActivityBinding = DataBindingUtil.setContentView<AboutActivityBinding>(this, R.layout.about_activity)
        setSupportActionBar(aboutActivityBinding.aboutToolbarNavigator)
        val actionBar = supportActionBar
        actionBar?.setDisplayHomeAsUpEnabled(true)

        title = "About Paperboy"
    }
}